app/src/test/java/org/mozilla/fenix/tabstray/DefaultTabsTrayControllerTest.kt

```diff
@@ -123,14 +122,14 @@ class DefaultTabsTrayControllerTest {
     fun `GIVEN private mode WHEN handleOpeningNewTab is called THEN Event#NewPrivateTabTapped is added to telemetry`() {
         createController().handleOpeningNewTab(true)
 
-        verify { metrics.track(Event.NewPrivateTabTapped) }
+        assertTrue(TabsTray.newPrivateTabTapped.testHasValue())
```

Comment:
nit: should we `assertFalse` on `testHasValue` before the controller call?  Apply to other cases as well.